projects/core/src/store-finder/occ/store-finder.service.ts

```diff
@@ -3,11 +3,12 @@ import { HttpClient, HttpParams } from '@angular/common/http';
 import { throwError, Observable } from 'rxjs';
 import { catchError, mergeMap } from 'rxjs/operators';
 
-import { SearchConfig } from '../../store-finder/models/search-config';
-import { LongitudeLatitude } from '../../store-finder/models/longitude-latitude';
+import { StoreFinderSearchConfig } from '../../store-finder/model/search-config';
+import { LongitudeLatitude } from '../../store-finder/model/longitude-latitude';
 
-import { OccConfig, StoreFinderSearchPage } from '@spartacus/core';
-import { OccE2eConfigurationService } from '../e2e/e2e-configuration-service';
+import { OccConfig } from '../../occ/config/occ-config';
+import { OccE2eConfigurationService } from '../occ/e2e/e2e-configuration-service';
+import { StoreFinderSearchPage } from '../../occ';
```

Comment:
I am not sure just `from '../../occ'` work well in AOT mode. https://github.com/angular/angular/issues/23713. We met this kind of errors before. So, to make it work well, we give explicitly path in the import. So, this one can be `import { StoreFinderSearchPage } from '../../occ/occ-models';`.

Reply:
Import changed.